src/i18n/el/translation.json

```diff
@@ -329,6 +329,7 @@
       "support": "Υποστήριξη",
       "privacy": "Απόρρητο",
       "termsOfService": "Όροι"
-    }
+    },
+    "noActivities": "Δεν υπάρχουν διαθέσιμες δραστηριότητες για να ολοκληρώσετε αυτήν τη στιγμή"
```

Comment:
I think this is meant to be `noApplets`, right? @ramirlm already had this in review https://github.com/ChildMindInstitute/mindlogger-web-refactor/pull/554

Reply:
No, this is a different ticket, actually. This is the empty state for an existing applet, but all activities are unassigned.